packages/assets-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [23.2.0]
+### Added
+- Add `getDefaultTokenListState` function to `TokenListController` ([#3744](https://github.com/MetaMask/core/pull/3744))
+- Add `getDefaultNftState` function to the `NftController` ([#3742](https://github.com/MetaMask/core/pull/3742))
+- Add `getDefaultTokensState` function to the `TokensController` ([#3743](https://github.com/MetaMask/core/pull/3743))
+
+### Changed
+- Price API perf improvements ([#3753](https://github.com/MetaMask/core/pull/3753), [#3755](https://github.com/MetaMask/core/pull/3755))
+  - Reduce token batch size from 100 to 30
+  - Sort token addresses in query params for more cache hits
+- Bump `@metamask/preferences-controller` to ^6.0.0
```

Comment:
Could you mark all of these `peerDependency` changes as breaking, and update the versions of each release including one to be a major version bump? All peer dependency bumps are breaking because they require changes to accommodate (i.e. the project using this package also needs to update the peer dependency)